data/slackModels/message.js

```diff
@@ -12,7 +12,8 @@ async function postMessage(message) {
 
 async function postOpenMessage(message) {
   try {
-    await axios.post(`${slack.baseUrl}/chat.postMessage`, qs.stringify(message));
+    const result = await axios.post(`${slack.baseUrl}/chat.postMessage`, qs.stringify(message));
```

Comment:
yo man, maybe after my pr is merged, you can pull from develop branch, resolve the possible merge conflict on this file before we push again